src/app/prevention/strides-against/game-config.ts

```diff
@@ -0,0 +1,86 @@
+import { Item } from './models/Item';
+import { BackgroundElement, ManElement } from './models/elements';
+
+export const BackgroundConfig: BackgroundElement = {
+  imageURL: '/assets/img/strides-against/background.jpg',
+  imageX: 0,
+  imageY: 0,
+  imageWidth: 800,
+  imageHeight: 450,
+};
+
+export const ManConfig: ManElement = {
+  imageURL: '/assets/img/strides-against/man.svg',
+  imageX: 90,
+  imageY: 92,
+  imageWidth: 25,
+  imageHeight: 50,
+};
+export const ObjectsConfig: Array<Item> = [
+  {
+    name: 'Garlic',
+    element: {
+      imageURL: '/assets/img/strides-against/garlic.png',
+      messageImageURL: '/assets/img/strides-against/messages/garlic.jpg',
+      messageBody: 'It\'s just a myth, consuming Garlic to prevent Malaria does not work.',
+      imageX: 425,
+      imageY: 100,
+      imageWidth: 40,
+      imageHeight: 40,
+      question: 'Consuming Garlic, helps prevent Malaria.',
+      answer: false,
+    }
+  }, {
+    name: 'RepellentCream',
+    element: {
+      imageURL: '/assets/img/strides-against/cream.png',
+      messageImageURL: '/assets/img/strides-against/messages/cream.png',
+      messageBody: ' Applying Mosquito repellent cream/spray help a lot.',
+      imageX: 592,
+      imageY: 180,
+      imageWidth: 45,
+      imageHeight: 70,
+      question: 'Applying Mosquito Repellent Cream helps prevent Malaria.',
+      answer: true,
+    }
+  }, {
+    name: 'AirCondition',
+    element: {
+      imageURL: '/assets/img/strides-against/air-conditioner.png',
+      messageImageURL: '/assets/img/strides-against/messages/air-conditioned.png',
+      messageBody: 'Staying in dry and air conditioned surroundings do help a lot',
+      imageX: 166,
+      imageY: 195,
+      imageWidth: 40,
+      imageHeight: 30,
+      question: 'Staying in dry air conditioned surroundings helps prevent Malaria.',
+      answer: true,
+    }
+  }, {
+    name: 'Clothes',
+    element: {
+      imageURL: '/assets/img/strides-against/clothes.png',
+      messageImageURL: '/assets/img/strides-against/messages/clothes.jpg',
+      messageBody: 'Always wear full shirts/pants to avoid bites by mosquitoes',
+      imageX: 340,
+      imageY: 290,
+      imageWidth: 40,
+      imageHeight: 40,
+      question: 'Wearing full sleeved clothes helps prevent Malaria.',
+      answer: true,
+    }
+  }, {
+    name: 'Homeopathy',
+    element: {
+      imageURL: '/assets/img/strides-against/homeopathy.png',
+      messageImageURL: '/assets/img/strides-against/messages/homeopathy.jpg',
+      messageBody: 'It\'s just a myth. Scientifically not proven. ',
```

Comment:
Can you get these strings from english.json file..

Reply:
I don't know but I may face difficulty in subscribing to Language service in a normal TS file given we don't have lifecycle hook in it. If language is a problem, we can have element objects in it for different languages and call from the component for a specific language. That would also help in keeping things related to it in its vicinity.